src/helm/proxy/clients/model_critique_client.py

```diff
@@ -14,6 +14,8 @@
 from helm.proxy.clients.client import Client
 from helm.proxy.clients.critique_client import CritiqueClient
 
+import anthropic
```

Comment:
Handle anthropic as an optional dependency like [this](https://github.com/stanford-crfm/helm/blob/v0.2.4/src/helm/proxy/clients/anthropic_client.py#L31-L35).